src/olympia/addons/serializers.py

```diff
@@ -300,7 +300,7 @@ def fake_version_object(self, obj, data):
             version = Version(
                 addon=obj, id=data['id'],
                 reviewed=self.handle_date(data['reviewed']),
-                version=data['version'])
+                version=data['version'], channel=data['channel'])
```

Comment:
We don't need channel _per se_ but the serializer does `get_url_path` on the (fake) Version object and that needs the channel property to be correctly set.  If there's an easier way that avoids all the ES reindexing I'll go with that.